### ISSUE TITLE
Enable type/docs label action for PRs

### ISSUE THREAD

**Issue by Eve832** [grafana/grafana#64251]
The docs team would like to enable the `type/docs` label action for PRs (in addition to issues) so we can better triage/ track open PRs.

**Comment by jdbaldry** [comment]
Unfortunately my PR only addressed half the problem.

@tolzhabayev I am considering adding the `labeled` type to the `on.pull_request` triggers for https://github.com/grafana/grafana/blob/main/.github/workflows/pr-commands.yml. I imagine this might add a significant number of workflows being run. Would you know if that would be a problem?

**Comment by tolzhabayev** [comment]
> Member

I don't think it will be a problem, they will mostly run once per PR as the concurrency will cancel them out -> e.g. when you add one label, then another one then another one - only the last one will run.

And since we already have [pr-checks running on label](https://github.com/grafana/grafana/blob/main/.github/workflows/pr-checks.yml), why not actions

**Comment by jdbaldry** [comment]
Awesome, thank you for reviewing this!

I'll submit the PR today.

**Comment by jdbaldry** [comment]
Eurgh: 

```console
Could not get project node id HttpError: Personal access tokens with fine grained access do not support the GraphQL API
```

**Comment by tolzhabayev** [comment]
yap yap, I think you need something like https://github.com/grafana/grafana/blob/main/.github/workflows/issue-opened.yml#L27

**Comment by jdbaldry** [comment]
no longer needed
